package.json

```diff
@@ -5,7 +5,7 @@
   "description": "InfluxDB broadcasting for Good process monitor",
   "main": "lib/index.js",
   "scripts": {
-    "test": "lab -m 5000 -t 79 -v -La code",
+    "test": "lab -m 5000 -t 92 -v -La code",
```

Comment:
We improved code coverage quite a bit.  Should we setup CI on this project?  I would be happy to try to wire in TravisCI or something to run on pull request creation, and to block merge if tests fail, have linting issues, have low code coverage, etc.